.changeset/big-ghosts-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cartesi/cli": patch
```

Comment:
in pre-release mode should we specify as major or as patch?
because it's meant to be `2.0.0-alpha.1`

Reply:
I'm considering this particular commit as a patch. I think everything that goes into prerelease will bump the last number after `alpha` only.